### PR TITLE
33 e2e testing

### PR DESCRIPTION
**Pull request by oonasaksman**
## Describe your changes

Implemented end to end testing for functionality that is now on the staging site. Creating a poll, navigating around the site and filling textfields in login and register.

## Checklist before requesting a review

-   [x] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [x] I have added thorough tests

### CODE REVIEW

**Review by nataajj — APPROVED** [review]

**Comment by Jontzii** [comment]
`submitPoll.cy.js` failed as I tried to run it

File: frontend/cypress/e2e/submitPoll.cy.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        const question = 'Is this working?';
+        cy.get('.question-field').type(`${question}`);
+    });
+
+    it('can choose to show vote count', () => {
+        cy.get('.vote-count-toggle-btn').click();
+        cy.get('.vote-count-toggle-btn').find('input').should('be.checked');
+    });
+
+    it('can submit poll', () => {
```
Comment (Jontzii) [inline frontend/cypress/e2e/submitPoll.cy.js:20]: This is failing!

**Review by Jontzii — CHANGES_REQUESTED** [review]
`submitPoll.cy.js` is failing, otherwise, everything looks good :)

**Comment by Jontzii** [comment]
Pushed fix to the pipeline and removed .mp4 created by cypress

**Review by Marakke — APPROVED** [review]

**Review by Jontzii — APPROVED** [review]

**Pull request merged**